guides/source/active_record_callbacks.md

```diff
@@ -161,6 +161,20 @@ Here is a list with all the available Active Record callbacks, listed in the sam
 [`around_update`]: https://api.rubyonrails.org/classes/ActiveRecord/Callbacks/ClassMethods.html#method-i-around_update
 [`before_update`]: https://api.rubyonrails.org/classes/ActiveRecord/Callbacks/ClassMethods.html#method-i-before_update
 
+```ruby
+class User < ApplicationRecord
+  around_update :audit_changes
+
+  private
+    def audit_changes
+      was_email = email_was
+      yield
+      log_changes = (was_email != email)
+      puts "Audit Log: Email changed from '#{was_email}' to '#{email}'" if log_changes
```

Comment:
prefer `Rails.logger.info` to `puts`. see suggestion above. `log_changes` should be renamed to `email_changed` instead.